source/SpinalHDL/Examples/Intermediates ones/vga.rst

```diff
@@ -192,20 +136,11 @@ Component and io definition
 
 Let's define a new VgaCtrl ``Component``\ , which takes as ``RgbConfig`` and ``timingsWidth`` as parameters. Let's give the bit width a default value of 12.
 
-.. code-block:: scala
-
-   class VgaCtrl(rgbConfig: RgbConfig, timingsWidth: Int = 12) extends Component {
-     val io = new Bundle {
-       val softReset = in Bool
-       val timings = in(VgaTimings(timingsWidth))
-       val pixels = slave Stream (Rgb(rgbConfig))
-
-       val error = out Bool
-       val frameStart = out Bool
-       val vga = master(Vga(rgbConfig))
-     }
-     ...
-   }
+.. literalinclude:: /../examples/src/main/scala/spinaldoc/examples/intermediate/VGA.scala
+   :language: scala
+   :start-at: case class VgaCtrl(
+   :end-before: // end VgaCtrl io
+   :append: ...
```

Comment:
Hmm indentation is missing here?

Reply:
You mean the indentation of the ellipsis? I tried a few thing, but none worked:
Using a literal text like
```
:append: `  ...`
```
get taken literally and introduces ` into the literal block

Defining a replacement via `.. |nbsp| unicode:: 0xA0 ` also gets taken literally. Looking at the code of literalinclude I'm not sure there is a way to do that since the text in append/prepend is not parsed.

Reply:
So I guess we can leave this as-is? Not a big issue I think